app/views/custom_styles/_interface.html.erb

```diff
@@ -0,0 +1,45 @@
+<%= form_tag update_design_colors_path, method: :post, class: "form" do %>
+  <section class="form--section">
+    <fieldset class="form--fieldset">
+      <legend class="form--fieldset-legend"><%= I18n.t(:label_interface_colours) %></legend>
+      <p> <%= t('label_interface_colours_description') %> </p>
```

Comment:
This here should be American English and therefore `label_interface_colors_description`. The current translation for this line is British, too. It also should be americanized to stay consistent. I've added a suggestion in the corresponding YAML file :)